docs/pages/versions/unversioned/sdk/calendar.md

```diff
@@ -522,14 +522,26 @@ A person or entity that is associated with an event by being invited or fulfilli
 
 ### RecurrenceRule
 
-A recurrence rule for events or reminders, allowing the same calendar item to recur multiple times.
+A recurrence rule for events or reminders, allowing the same calendar item to recur multiple times. This interface is
+based on [the iOS interface](https://developer.apple.com/documentation/eventkit/ekrecurrencerule/1507320-initrecurrencewithfrequency) which is in turn based on
+[the iCal RFC](https://tools.ietf.org/html/rfc5545#section-3.8.5.3) so you can refer to those to learn more about this
+potentially complex interface.
+
+Not all of the combinations make sense. For example, when frequency is `DAILY`, setting `daysOfTheMonth` makes no sense.
```

Comment:
Be more precise about the platform. You use here `daysOfTheMonth` which isn't available on Android. Signal to users that Android doesn't have every option yet.

Reply:
Each of the added "complex" options is marked as both optional and iOS only, I think it's already descriptive enough, no?

Reply:
Ok. It's enough :)